File: Basic Programs/TypeScript/areaOfCircle.ts
function areaOfCircle (radius:number):number|string {
    if(radius <= 0){
        return "invalid radius"
    }
    const area:number = Math.PI * Math.pow(radius , 2);
    return area;
    
    }
    const radius:number = 5;
    const  area:number|string = areaOfCircle(radius)
    console.log(area);
    

File: Basic Programs/java/Areaofcircle.java

public class Areaofcircle{
public static void main(String[] args) { 
    int radius = 0;
    double area = areaofcircle(radius);
    System.out.println(area);
}

public static double areaofcircle(int radius){
    if(radius  <= 0){
        System.out.println("enter the correct radius");
    }
    double area = Math.PI * Math.pow(radius, 2);

      return area;
}


} 